packages/cli/src/controller/init-controller.ts

```diff
@@ -172,40 +174,44 @@ export async function prepare(projectPath: string, project: ProjectSpecBase): Pr
   }
 }
 
-async function preparePackage(projectPath: string, project: ProjectSpecBase): Promise<void> {
+export async function preparePackage(projectPath: string, project: ProjectSpecBase): Promise<void> {
   //load and write package.json
   const packageData = await fs.promises.readFile(`${projectPath}/package.json`);
   const currentPackage = JSON.parse(packageData.toString());
   currentPackage.name = project.name;
-  currentPackage.version = project.version;
   currentPackage.description = project.description ?? currentPackage.description;
   currentPackage.author = project.author;
-  currentPackage.license = project.license;
   const newPackage = JSON.stringify(currentPackage, null, 2);
   await fs.promises.writeFile(`${projectPath}/package.json`, newPackage, 'utf8');
 }
 
-async function prepareManifest(projectPath: string, project: ProjectSpecBase): Promise<void> {
-  //load and write manifest(project.yaml)
+export async function prepareManifest(projectPath: string, project: ProjectSpecBase): Promise<void> {
+  //load and write manifest(project.ts/project.yaml)
+  const tsPath = path.join(`${projectPath}`, `project.ts`);
```

Comment:
I think there's a constant for `project.ts`. Can you please update all occurrences to use that